src/components/Footer/Footer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Logo from '../Logo/Logo'
+import './Footer.css';
+
+const Footer = () => {
+    return (
+       <footer>
+           <h4>FaceIt</h4> 
+             <h5>is brought to you by</h5>
+              <div className="dev-container">
+                <div className="dev">
+                  <p>Kelsie Besinger Yeh</p>
+                  <div className="links">
+                    <a href="https://github.com/kelsiebesingeryeh/">Github</a>
```

Comment:
What do you think about having icons/svgs instead of just plain links? I like how a lot of websites use that.

Reply:
agreed, it is pretty easy to do! I did this for my personal website

Reply:
Yes! I was actually intending to do that originally but went down a rabbit hole looking at icons, so I just put the links up as a placeholder for now and forgot about it.  Thanks for the reminder!